### PR TITLE
Ensure to press next in nis_server

### PR DESCRIPTION
**Pull request by tinawang123**
Ensure to press next in nis_server

- Related ticket: https://progress.opensuse.org/issues/124583
- Verification run: https://openqa.suse.de/tests/10547482#

### CODE REVIEW

File: tests/x11/nis_server.pm [hunk 1]
```diff
@@ -68,7 +68,7 @@ sub nis_server_configuration {
     assert_screen 'nis-server-server-maps-setup-finished';
     send_key $cmd{next};
     # NIS Server Query Hosts
-    assert_screen 'nis-server-query-hosts-setup';
+    apply_workaround_bsc1204176('nis-server-query-hosts-setup') if (is_sle('>=15-SP4'));
```
Comment (lemon-suse) [inline tests/x11/nis_server.pm:71]: What will be there if VERSION less than 15-SP4?

Comment (tinawang123) [reply tests/x11/nis_server.pm:71]: It is related with bsc1204176. It doesn't have this problem except 15sp5.

Comment (lemon-suse) [reply tests/x11/nis_server.pm:71]: Ok, so this should need workaround for bsc1204176.

**Review by lemon-suse — APPROVED** [review]

**Review by chcao — APPROVED** [review]
LGTM

**Pull request merged**